pkg/cmd/server/start/node_args.go

```diff
@@ -42,6 +44,7 @@ func BindNodeArgs(args *NodeArgs, flags *pflag.FlagSet, prefix string) {
 	flags.StringVar(&args.VolumeDir, prefix+"volume-dir", "openshift.local.volumes", "The volume storage directory.")
 	// TODO rename this node-name and recommend hostname -f
 	flags.StringVar(&args.NodeName, prefix+"hostname", args.NodeName, "The hostname to identify this node with the master.")
+	flags.StringVar(&args.NetworkPluginName, prefix+"network-plugin", args.NetworkPluginName, "The network plugin to be called for configuring networking for pods.")
```

Comment:
Network plugin is probably an important flag.  How many other options would it have?


> On Apr 9, 2015, at 8:12 PM, Rajat Chopra <notifications@github.com> wrote:
> 
> In pkg/cmd/server/start/node_args.go:
> 
> > @@ -42,6 +44,7 @@ func BindNodeArgs(args *NodeArgs, flags *pflag.FlagSet, prefix string) {
> >  	flags.StringVar(&args.VolumeDir, prefix+"volume-dir", "openshift.local.volumes", "The volume storage directory.")
> >  	// TODO rename this node-name and recommend hostname -f
> >  	flags.StringVar(&args.NodeName, prefix+"hostname", args.NodeName, "The hostname to identify this node with the master.")
> > +	flags.StringVar(&args.NetworkPluginName, prefix+"network-plugin", args.NetworkPluginName, "The network plugin to be called for configuring networking for pods.")
> Not stuck on it. Can definitely remove it. What if someone wants to specify it through the command line, though, just like volume-dir?
> @smarterclayton ?
> 
> —
> Reply to this email directly or view it on GitHub.
>